index.js

```diff
@@ -69,63 +72,124 @@ app.post('/chargeForCookie', async (request, response) => {
       `[Error] Status:${e.status}, Messages: ${JSON.stringify((JSON.parse(e.response.text)).errors, null, 2)}`);
 
     const { errors } = (JSON.parse(e.response.text));
+    sendErrorMessage(errors, response);
+  }
+});
 
-    switch(errors[0].code) {
-        case "UNAUTHORIZED":
-          response.status(401).send({
-              errorMessage: "Server Not Authorized. Please check your server permission."
-          });
-          break;
-        case "GENERIC_DECLINE":
-          response.status(400).send({
-              errorMessage: "Card declined. Please re-enter card information."
-          });
-          break;
-        case "CVV_FAILURE":
-          response.status(400).send({
-              errorMessage: "Invalid CVV. Please re-enter card information."
-          });
-          break;
-        case "ADDRESS_VERIFICATION_FAILURE":
-          response.status(400).send({
-              errorMessage: "Invalid Postal Code. Please re-enter card information."
-          });
-          break;
-        case "EXPIRATION_FAILURE":
-          response.status(400).send({
-              errorMessage: "Invalid expiration date. Please re-enter card information."
-          });
-          break;
-        case "INSUFFICIENT_FUNDS":
-          response.status(400).send({
-              errorMessage: "Insufficient funds; Please try re-entering card details."
-          });
-          break;
-        case "CARD_NOT_SUPPORTED":
-          response.status(400).send({
-              errorMessage: "	The card is not supported either in the geographic region or by the MCC; Please try re-entering card details."
-          });
-          break;
-        case "PAYMENT_LIMIT_EXCEEDED":
-          response.status(400).send({
-              errorMessage: "Processing limit for this merchant; Please try re-entering card details."
-          });
-          break;
-        case "TEMPORARY_ERROR":
-          response.status(500).send({
-              errorMessage: "Unknown temporary error; please try again;"
-          });
-          break;
-        default:
-          response.status(400).send({
-              errorMessage: "Payment error. Please contact support if issue persists."
-          });
-          break;
-    }
+app.post('/chargeCustomerCard', async (request, response) => {
+  const requestBody = request.body;
+  const locations = await locationsApi.listLocations();
```

Comment:
Suggest putting this in a try block as well

Reply:
I accidentally merged before it loaded these comments, but do you mean to move this down in the below try block, or into its own?

(I will be creating another PR for these changes)

Reply:
Nvm, I see how it is in the other endpoint. I can refactor it to match